src/Unicorn/Configuration/XmlConfigurationProvider.cs

```diff
@@ -87,6 +88,7 @@ protected virtual IConfiguration LoadConfiguration(XmlElement configuration, Xml
 				{"evaluator", RegisterExpectedConfigType<IEvaluator>},
 				{"predicate", RegisterExpectedConfigType<IPredicate>},
 				{"targetDataStore", RegisterExpectedConfigType<ITargetDataStore>},
+				{"sourceControlSync", RegisterExpectedConfigType<ISourceControlSync>},
```

Comment:
It won't be necessary to register this as an expected configuration, that's for "stuff Unicorn can't live without"

Normal registration should be enough, no?